### PR TITLE
[DOC-3056] Update SMP monitoring topic

### PR DESCRIPTION
**Pull request by brian-f-harness**
Update SMP monitoring topic

For reviewers, [a preview](https://645113e1b7303106a4a22163--harness-developer.netlify.app/docs/self-managed-enterprise-edition/monitor-self-managed-enterprise-edition/monitor-harness-on-prem) is available.

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [x] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [x] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

**Review by kapilgarg1996 — DISMISSED** [review]

File: docs/self-managed-enterprise-edition/monitor-self-managed-enterprise-edition/monitor-harness-on-prem.md [hunk 1]
```diff
@@ -32,54 +32,33 @@ This example setup requires:
...
-- Istio version 1.15.3 or Nginx version v1.0.0-alpha.2
+- Istio version 1.15.3
+- Nginx version v1.0.0-alpha.2
 
 :::info note
 For this example, we use the Prometheus operator packaged by Bitnami as an external Prometheus setup.
 :::
 
-## Install an ingress controller
+## Configure metrics and Ingress rules
```
Comment (StrawberryPad) [inline docs/self-managed-enterprise-edition/monitor-self-managed-enterprise-edition/monitor-harness-on-prem.md:42]:
```suggestion
## Configure metrics and ingress rules
```

File: docs/self-managed-enterprise-edition/monitor-self-managed-enterprise-edition/monitor-harness-on-prem.md [hunk 2]
```diff
@@ -32,54 +32,33 @@ This example setup requires:
...
-  :::
-
-2. Confirm your `ingress-controller` deployment is up and running to create a service with an external IP address. Harness recommends that you set up the controller in the same namespace where your deploy your Harness services.
-
-  :::info note
-  You must enable metrics and the `serviceMonitors` for your databases to view the services exposing the metrics for each database. 
-  :::
-
-3. Create an ingress file for `mongo-metrics`, with defined routing rules that forwards requests to an internal service exposing metrics with a similar configuration.
+2. Create an ingress file for `metrics`, with defined routing rules that forwards requests to an internal service exposing metrics with a similar configuration.
```
Comment (StrawberryPad) [inline docs/self-managed-enterprise-edition/monitor-self-managed-enterprise-edition/monitor-harness-on-prem.md:61]:
```suggestion
2. Create an ingress file for `metrics`, with defined routing rules that forward requests to an internal service exposing metrics with a similar configuration.
```

File: docs/self-managed-enterprise-edition/monitor-self-managed-enterprise-edition/monitor-harness-on-prem.md [hunk 3]
```diff
@@ -107,9 +86,9 @@ Follow the steps below on the Kubernetes cluster where you deploy your Harness i
    Add your IPs to your allow list so the metrics exposed by the ingress are only accessible internally. The IP included in the allow list is the external IP for the node where you host Prometheus in a separate cluster.
   :::
 
-## Deploy Prometheus to integrate with Harness
+## Configure Prometheus to integrate with Harness
 
-There are two options you can use to deploy Prometheus to integrate with your Harness instance:
+Follow the instructions below based on the type of your prometheus installation to integrate with your Harness instance:
```
Comment (StrawberryPad) [inline docs/self-managed-enterprise-edition/monitor-self-managed-enterprise-edition/monitor-harness-on-prem.md:91]:
```suggestion
To integrate Prometheus with your Harness instance, follow the instructions below for your type of prometheus installation:
```

Comment (StrawberryPad) [reply docs/self-managed-enterprise-edition/monitor-self-managed-enterprise-edition/monitor-harness-on-prem.md:91]: I thought the original sentence was a bit awkward when spoken.

**Review by StrawberryPad — CHANGES_REQUESTED** [review]

**Review by StrawberryPad — APPROVED** [review]

**Pull request merged**